iam_check/test/test_accessAnalyzer.py

```diff
@@ -18,6 +18,16 @@ def test_a2_policy_validation(self):
         findings = _load_json_file("test/iam_policy/findings.json")
         assert(Reporter(None, None, None).build_report_from(check.findings).to_json() == findings)
 
+    def test_a2_code_blocking(self):
+        file = _load_json_file(f"test/multiple_policies/test_plan.json")
+        plan = TerraformPlan(**file)
+        reference_policy = _load_json_file(f"test/multiple_policies/identity_codes_reference_policy.json")
+        check = Validator("123456789012", "us-west-2", "aws")
+        check.run(plan)
+        test_findings = _load_json_file(f"test/multiple_policies/identity_codes.json")
+        report = Reporter(None, [], None, ['ALLOW_WITH_UNSUPPORTED_TAG_CONDITION_KEY_FOR_SERVICE']).build_report_from(check.findings).to_json()
```

Comment:
This was pretty difficult to nail down the exact message. not sure how precise we need it but this allows us to validate that the FINDING CODE does indeed get added to the BreakingFindings